scanpy/experimental/pp/_recipes.py

```diff
@@ -0,0 +1,159 @@
+from typing import Optional, Tuple
+from anndata import AnnData
+import pandas as pd
+import numpy as np
+from scanpy import experimental
+from scanpy.preprocessing import pca
+
+
+def recipe_pearson_residuals(
+    adata: AnnData,
+    theta: float = 100,
```

Comment:
```suggestion
    adata: AnnData,
    *,
    theta: float = 100,
```

Should we make all arguments key-word only like for the internally used functions?

Reply:
added, also for `normalize_pearson_residuals_pca` where it was still missing